### PR TITLE
f/aws_organizations_policy:support for tags

### LINKED ISSUE #15168: resource/aws_organizations_policy: Add support for resource tags
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS Organizations Policies support resource tags. Adding and updating tags on a policy is supported via the AWS Organizations API with the `CreatePolicy` and `TagResource` API calls.

Terraform doesn't currently support resource tags for the `aws_organizations_policy` resource type but it would be good if it did. This would extend the degree to which these resources can be managed programmatically.

### Affected Resource(s)

- `aws_organizations_policy`

### Potential Terraform Configuration

```hcl
resource "aws_organizations_policy" "service_control_policy" {
  name        = "Sample policy"
  description = "Sample policy description"
  content     = <<CONTENT
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "RequireMicroInstanceType",
      "Effect": "Deny",
      "Action": "ec2:RunInstances",
      "Resource": "arn:aws:ec2:*:*:instance/*",
      "Condition": {
        "StringNotEquals":{               	
          "ec2:InstanceType":"t2.micro"
        }
      }
    }
  ]
}
CONTENT

  tags = {
    Environment = "Production"
    Division    = "Operations"
  }
}
```

### Expected Behavior

The AWS Organizations policy should be created or updated with the tags specified.

### Actual Behavior

Terraform returns the following `Unsupported argument` error:

```
Error: Unsupported argument

  on modules/service_control_policy/main.tf line 23, in resource "aws_organizations_policy" "service_control_policy":
   23:   tags = {

An argument named "tags" is not expected here.
```

### References

- [AWS Organizations API Reference - CreatePolicy](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreatePolicy.html)
- [AWS Organizations API Reference - TagResource](https://docs.aws.amazon.com/organizations/latest/APIReference/API_TagResource.html)

### PR DESCRIPTION
**Pull request by nikhil-goenka**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #15168

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note

```

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=testAccAwsOrganizationsPolicy_tags'
--- PASS: TestAccAwsOrganizationsPolicy_tags (164.96s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       165.341s
...
```

### CODE REVIEW

File: aws/resource_aws_organizations_policy_test.go [hunk 1]
```diff
@@ -113,6 +113,51 @@ func testAccAwsOrganizationsPolicy_description(t *testing.T) {
...
+				),
+			},
+			{
+				Config: testAccAwsOrganizationsPolicyConfig_TagC(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsOrganizationsPolicyExists(resourceName, &p3),
+                    resource.TestCheckResourceAttr(resourceName, "tags.%", "1"),
+					resource.TestCheckResourceAttr(resourceName, "tags.TerraformProviderAwsTest", "true"),
+				),
+			},
```
Comment (anGie44) [inline aws/resource_aws_organizations_policy_test.go:156]: Let's add 1 final step here with a config (could use `testAccAwsOrganizationsPolicyConfig_Required`) that doesn't have `tags` configured to verify we can completely remove the tags and check that the count is 0

**Review by anGie44 — CHANGES_REQUESTED** [review]
Hi @nikhil-goenka, thank you for this PR! It's looking pretty great, just note you'll need to run `go fmt` and add the new test to `aws/resource_aws_organizations_test.go` within the  `"Policy"` test case section to remedy the CI checks.

File: aws/resource_aws_organizations_policy_test.go (hunk 1 above)
Comment (nikhil-goenka) [reply aws/resource_aws_organizations_policy_test.go:156]: Added

**Review by anGie44 — APPROVED** [review]
Looks great, thanks @nikhil-goenka 🚀 

Tests passing except for known failure in `TestAccAWSOrganizations_serial/Policy/Type_Backup`
Output of acceptance tests:
```
    --- PASS: TestAccAWSOrganizations_serial/Organization (299.23s)
        --- PASS: TestAccAWSOrganizations_serial/Organization/basic (24.35s)
        --- PASS: TestAccAWSOrganizations_serial/Organization/AwsServiceAccessPrincipals (57.70s)
        --- PASS: TestAccAWSOrganizations_serial/Organization/EnabledPolicyTypes (153.75s)
        --- PASS: TestAccAWSOrganizations_serial/Organization/FeatureSet (19.60s)
        --- PASS: TestAccAWSOrganizations_serial/Organization/DataSource (43.84s)
    --- PASS: TestAccAWSOrganizations_serial/Account (0.00s)
        --- SKIP: TestAccAWSOrganizations_serial/Account/basic (0.00s)
        --- SKIP: TestAccAWSOrganizations_serial/Account/ParentId (0.00s)
        --- SKIP: TestAccAWSOrganizations_serial/Account/Tags (0.00s)
    --- PASS: TestAccAWSOrganizations_serial/OrganizationalUnit (65.23s)
        --- PASS: TestAccAWSOrganizations_serial/OrganizationalUnit/basic (23.85s)
        --- PASS: TestAccAWSOrganizations_serial/OrganizationalUnit/Name (41.38s)
    --- PASS: TestAccAWSOrganizations_serial/OrganizationalUnits (0.00s)
        --- PASS: TestAccAWSOrganizations_serial/OrganizationalUnits/DataSource (21.95s)
    --- FAIL: TestAccAWSOrganizations_serial/Policy (280.93s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/concurrent (30.34s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/Description (40.31s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/Tags (74.96s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/Type_AI_OPT_OUT (22.35s)
        --- FAIL: TestAccAWSOrganizations_serial/Policy/Type_Backup (10.29s) -- current failure in master
        --- PASS: TestAccAWSOrganizations_serial/Policy/Type_SCP (38.88s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/Type_Tag (22.21s)
        --- PASS: TestAccAWSOrganizations_serial/Policy/basic (41.59s)
    --- PASS: TestAccAWSOrganizations_serial/PolicyAttachment (113.89s)
        --- PASS: TestAccAWSOrganizations_serial/PolicyAttachment/Account (50.50s)
        --- PASS: TestAccAWSOrganizations_serial/PolicyAttachment/OrganizationalUnit (32.60s)
        --- PASS: TestAccAWSOrganizations_serial/PolicyAttachment/Root (30.79s)
```

**Pull request merged**